ui/apps/platform/src/Containers/Workflow/WorkflowEntityPage.js

```diff
@@ -23,6 +23,12 @@ const useCaseDefaultSorts = {
     [useCases.VULN_MANAGEMENT]: vulnMgmtDefaultSorts,
 };
 
+function removeGraphqlAlias(fieldName) {
+    const parts = fieldName?.split(':');
+
+    return parts ? parts[0] : '';
```

Comment:
If `parts` is a falsey value, wouldn't that break things?

Reply:
Falsey will fall through to the empty string option, which I only observed right before the component dismounts before returning to a list page.